### DISCUSSION TITLE
Behavior combining 'Sailing Mode' and 'Disable Sleep Until Charge Limit'

### DISCUSSION THREAD

**Discussion by michelro** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#480] [Q&A] [answered]
Hi there!

Just wondering how 'sailing mode' and 'disable sleep until charge limit' work together.

Consider the scenario where charge limit is set to 70%, sailing mode enabled, difference set at 5%.
Laptop is plugged in to the power source and the  battery level is at 67%, as an example.

At this point the battery will drain until it reaches 65% before it starts to charge again to 70%.
I'm sailing to 65%, pretty slowly, as the main source of power is my power adapter.

If 'disable sleep until charge limit' is turned on, and I close the lid, what will happen?
Will my macbook be awaken for ages, until the battery goes down naturally to 65% and charge again to 70%?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @michelro,

The feature Disable Sleep Until Charge Limit will disable sleep only when your MacBook is charging. When your MacBooks SoC is lower than the Charge Limit but charging is paused because of the Sailing Mode, sleep will not be disabled.

Cheers,
Matthias
